mmf/models/movie_mcan.py

```diff
@@ -80,7 +80,7 @@ def _init_feature_encoders(self, attr: str):
         with omegaconf.open_dict(feat_encoder_config):
             feat_encoder_config.params.model_data_dir = self.config.model_data_dir
             feat_encoder_config.params.in_dim = feature_dim
-        feat_model = build_image_encoder(feat_encoder_config, direct_features=True)
+        feat_model = build_image_encoder(feat_encoder_config, direct_features=False)
```

Comment:
Will this still support the older encoder? Otherwise, it would make sense to rely on `build_encoder` function.

Reply:
There was no old encoder. It was using the feature from sample list directly. This won't change that behavior.